src/lib/contributers.js

```diff
@@ -26,6 +27,10 @@ const useContributerData = props => {
                 })
             })
 
+            contribtorList = filter(contribtorList, el => {
+                return el.username !== 'anushbhatia' && el.username !== 'smaranjitghose' && el.username !== 'abhu-A-J'
+            })
+
```

Comment:
Filter it right here by replacing map with filter. 
Use .includes methods for filtering condition. 
MDN : https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/includes

Also make a folder in src with name data and add two JS files each exploring a static array of objects for admins and mentors . Fill in these details manually with same information you get from API

Reply:
I had installed the loadash then realized that I could do it without it. I am reverting the code and filter in map function only

Reply:
Do the second part in here as well for the issue for getting a list of admins and mentors!